cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -633,6 +635,10 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
                 result.templateInitContainerSecurityContext = template.getInitContainer().getSecurityContext();
             }
 
+            if (template.getClusterCaCert() != null && template.getClusterCaCert().getMetadata() != null) {
+                result.templateClusterCACertLabels = template.getClusterCaCert().getMetadata().getLabels();
+                result.templateClusterCACertAnnotations = template.getClusterCaCert().getMetadata().getAnnotations();
+            }
```

Comment:
These variables do not seem to be used anywhere. So I think it could be just removed?